CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# See https://github.blog/2017-07-06-introducing-code-owners/ for details
+# Format of the file follows: https://git-scm.com/docs/gitattributes
+
+# Fallback to core group for crates or folders without specific owners, like docs/.
+
+* @nciric @zbraniecki @sffc @echeran
```

Comment:
Is there a way to put a group here instead of individual usernames?

Reply:
Yes, something like @org/team-name
We need the group for crate.io too.

We can tweak this file after I land it.

Reply:
I think you might be able to use `@unicode-org/icu4x-sc`.

> The pattern is followed by one or more GitHub usernames or team names using the standard @username or @org/team-name format. 

https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax

Reply:
I changed to what @kpozin proposed, but I wanted more focused team/group there. It seems I can't create a team in our repo. Is there icu4x org? I am member of omnicu placeholders...